.github/workflows/pull_request.yml

```diff
@@ -325,7 +325,7 @@ jobs:
         id: commit
         run: |
           branch_name='preview-${{ github.event.pull_request.number }}'
-          git diff origin/main -- docs >/dev/null && changes=true || changes=false
```

Comment:
This was the problem is it was looking for a `docs` folder which is not correct on this repo. So, because of this it was perpetually saying that there were no changes (aka `changes=false`) because it was comparing a folder which didn't exist on this repo. And if `changes=false` the comment is not printed.